azerothcore/azerothcore-wotlk
# fix(DB/Loot): removes under-leveled loot from Anvilrage Guardsman

## Changes Proposed:
-  Removes a multiple drops from Avilrage Guardsman loot table, in accordance with addressed issue
-  Removes Citrine from loot table 24036

## Issues Addressed:

- Closes #8034 
- Closes https://github.com/chromiecraft/chromiecraft/issues/1815

## SOURCE:

Note for removal of Citrine: This reference_loot_template entry has multiple (1000+) different mobs referencing it.
Checked a couple of the mobs (On Wowhead) and found that it varied whether they dropped the item (Citrine)
Those that did had it in their individual table e.g. [Moshogg Warmonger](https://tbc.wowhead.com/npc=709/moshogg-warmonger#drops) 


## Tests Performed:

- Done as below


## How to Test the Changes:


1. Run SQL query — Should change respectivly 13 and 1 row
2. Check that Anvilrage Guardsman doesn't drop any of the listed items in the addressed issue
     * Through creature_loot_template for 12 of the items
     * Through reference_loot_template for Citrine

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
